vulnerabilities/importers/openssl.py

```diff
@@ -49,27 +50,13 @@ def updated_advisories(self) -> Set[Advisory]:
         # (url, etag) mappings in the DB. `create_etag`  creates
         # (url, etag) pair. If a (url, etag) already exists then the code
         # skips processing the response further to avoid duplicate work
-        if self.create_etag(self.url):
+        if create_etag(data_src=self, url=self.url, etag_key="ETag"):
```

Comment:
That makes sense. I want to follow the semantics of Django ORM here, does `get_or_create_etag` make more sense ?

Reply:
I just realized  what I just said was stupid :) . `create_etag` makes sense since the function returns True only if the etag is created otherwise that's gonna return Falsie.  It only returns the info required to deduce whether the encountered etag is new or not.